Automattic/social-logos
# Icon Font generation: only generate an embedded woff2 font.

* We do not really need ttf or woff files, since woff2 is widely supported these days. The only practical thing also using WOFF buys you is Internet Explorer 11 support, which we do not need to support these days.
* While loading an external woff2 file works in a demo, it can cause some issues on sites using a CDN to load some of the assets.
* Let's update the CSS class prefixes to match what's used in Automattic products.## Proposed changes:

In this PR, we're changing the way the icon font is generated in this package. From now on, **we'll only generate an embedded woff2 font** instead of shipping multiple font files and loading them via font-face.

### Reasoning

**WOFF2 browser support**

We do not really need ttf or woff files, since woff2 is widely supported  these days, and typically smaller (and thus more performant) than the other older font formats. The only practical thing also using WOFF buys you is Internet Explorer 11 support, which we do not need to support these days.

Reference: https://caniuse.com/woff2

**Issues when loading external font files**

While loading an external woff2 file works in a demo, it can cause some
issues on sites using a CDN to load some of the assets.

**********

On top of this, I'm changing the CSS class prefixes to match what we use in other Automattic products.

> **Note**
> Because of all these changes, I'm doing a minor version bump for this package.

## Product discussion:

* p9dueE-7DU-p2

## Testing instructions:

* Load the demo file in your browser
* Check that each icon is displayed properly.
